src/gam/gapi/cloudidentity/devices.py

```diff
@@ -129,11 +129,26 @@ def print_():
         if myarg in ['filter', 'query']:
           device_filter = sys.argv[i+1]
           i += 2
+        elif myarg == 'view':
```

Comment:
This just seems like another way of doing the same thing nocompanydevices/nopersonaldevices does. What's reason for adding it?